### DISCUSSION TITLE
How can I find this file locally?

### DISCUSSION THREAD

**Discussion by linghuccc** [rainbow-me/rainbowkit#2104] [Q&A]
The file URL is https://github.com/rainbow-me/rainbowkit/blob/b25db9a1f1f5acc96e28a12e38c2156e4fdbf51d/examples/with-create-react-app/src/polyfills.ts

It seems to be on the branch "b25db9a" of rainbow-me/rainbowkit git directory. But when I click on the drop down box, there is no branch "b25db9a". (I wanted to paste a picture here, but it seems that it is not allowed.)

I can download the repository to local but I am not able to find the file if I do not know which branch it is on.

Please advise. Thank you.

**Comment by DanielSinclair** [comment]
This is an example; feel free to copy and paste into your local project.
